docs/site/reference.md

```diff
@@ -119,7 +119,7 @@ patient data and used during query retrieval. The following blocking key types a
 
 `IDENTIFIER` (ID: **10**)
 
-:  The identifier triplet containing only the type, authority, and last 4 digits of the value
+:  A colon separated string of the identifier type, first 2 characters of the authority and last 4 characters of the value.
```

Comment:
For my own understanding, are the types limited to the types specified in the FHIR docs?

Reply:
They are, the link is in the reference guide (just seeing now that its a string and not a hyperlink so I'll make a small adjustment to fix that).  https://cdcgov.github.io/RecordLinker/reference.html#features

But here is the URL to all the available types http://hl7.org/fhir/R4/v2/0203/index.html